### PR TITLE
Added Circular Doubly Linked List Using C ( Fixes issue #1777)

### LINKED ISSUE #1777: Circular Doubly Linked List Using C
I would like to add Circular Doubly Linked List Code in C language. Please assign me for the task as a part of GSSoC20. @mrwonko @poush @sapanshah @alculquicondor @norvinyecla

### PR DESCRIPTION
**Pull request by SohelRaja**
Fixes #1777

#### Checklist:
- [x] Tab indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Circular Doubly Linked List Using C to the Circular_Doubly_Linked_List Directory. 

#### Languages Used:
- C

#### Files Added:
- Circular_Doubly_Linked_List.c

> We're happy to help you get this ready -- don't be afraid to ask for help.
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia @jainaman224 please review the PR.
Thanks!

### CODE REVIEW

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.c [hunk 1]
```diff
@@ -0,0 +1,441 @@
+#include<stdio.h>
```
Comment (MastersAbh) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.c]: Please add problem desc at start of file

File: Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.c [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+		do
+		{
+			printf("%d\t", ptr->data);
+			ptr = ptr->next;
+		}while(ptr != start);
+	}
+	option();	
+}
+
+//insert_beg function
```
Comment (MastersAbh) [inline Circular_Doubly_Linked_List/Circular_Doubly_Linked_List.c]: Please use 4 space indentation across the file.
I see TAB has been used here

**Comment by SohelRaja** [comment]
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia @jainaman224

**Comment by MastersAbh** [comment]
@SohelRaja I also see 2 other files in this PR. Please remove those files from the commits.

**Comment by MastersAbh** [comment]
@SohelRaja are you working on this ?

**Comment by SohelRaja** [comment]
Yes I am working on it.

**Comment by MastersAbh** [comment]
@SohelRaja I dont see any update for 11 days, Please close this PR if not working on it.

**Comment by SohelRaja** [comment]
> @SohelRaja I also see 2 other files in this PR. Please remove those files from the commits.

Problem solved

**Pull request closed without merging**